test/services/validation/CSOPTestData/ERSValidationCSOPRCLTestData.scala

```diff
@@ -80,22 +80,22 @@ trait ERSValidationCSOPRCLTestData {
       Cell("C", rowNumber, "Ten"),
       Cell("C", rowNumber, "123456789012345.1234"),
       Cell("D", rowNumber, "John"),
-      Cell("D", rowNumber, " "),
+      Cell("D", rowNumber, "AbcdefghijklmnopqrstuvwxyzAbcdefghijklmnopqrstuvwxyz"),
       Cell("D", rowNumber, "AbcdefghijklmnopqrstuvwxyzAbcdefghijklmnopqrstuvwxyz"),
       Cell("E", rowNumber, "John"),
       Cell("E", rowNumber, "AbcdefghijklmnopqrstuvwxyzAbcdefghijklmnopqrstuvwxyz"),
       Cell("F", rowNumber, "John"),
-      Cell("F", rowNumber, ""),
+      Cell("F", rowNumber, "AbcdefghijklmnopqrstuvwxyzAbcdefghijklmnopqrstuvwxyz"),
       Cell("F", rowNumber, "AbcdefghijklmnopqrstuvwxyzAbcdefghijklmnopqrstuvwxyz"),
       Cell("G", rowNumber, "AB123456A"),
       Cell("G", rowNumber, "ABB25345BA1"),
       Cell("G", rowNumber, "AB123456"),
-      Cell("G", rowNumber, ""),
+      Cell("G", rowNumber, "AB123456"),
       Cell("H", rowNumber, "123/XZ55555555"),
       Cell("H", rowNumber, "1234/12345/67890abcd"),
-      Cell("H", rowNumber, ""),
+      Cell("H", rowNumber, "1234/12345/67890abcd"),
       Cell("I", rowNumber, "Yes"),
-      Cell("I", rowNumber, " "),
+      Cell("I", rowNumber, "YyEeSs"),
```

Comment:
I don't think the changes from `" "` and  `""` are necessary, in this example we now just have two rows with `YyEeSs`?

Reply:
@harish-korada1 I still think you need the test case with an empty string for example, I dont think we should just delete them.